outdated content about web sites/webjobs

SychevIgor opened the pull request:
webjobs were released few years ago 
azure web sites for a multiple years are - azure app services web apps.

azurecla commented:
Hi __@ronortloff__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@ronortloff : Thanks for your contribution to the Azure documentation! The author, @BYHAM, has been notified to review your proposed change.

tysonn commented:
Closing - please see [Article authors working in the public repo](https://microsoft.sharepoint.com/teams/azurecontentguidance/article-author-working-in-the-public-repo).

SychevIgor committed:
outdated content

webjobs were released few years ago 
azure web sites for a multiple years are - azure app services web apps.
